dotnet/README.md

```diff
@@ -0,0 +1,49 @@
+# Executing WebAssembly modules with .NET
+
+This directory shows how a WebAssembly module can be executed by a [.NET Core](https://dotnet.microsoft.com/) program.
+
+While the Wasmtime for .NET API does not yet support the interface types proposal, this demonstrates how the [markdown WebAssembly module](https://github.com/bytecodealliance/wasmtime-demos/tree/master/markdown) can still be used from .NET.
```

Comment:
My main suggestion here is that we could specifically point out that interface types will make this case of passing a string between wasm and C# much smoother, since, looking at the code, that's the thing which stands out as being particularly awkward.

Reply:
The upcoming blog post will make this abundantly clear and I wanted to minimize the amount of duplicated content.

But I can expand upon this some.